airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/streams.py

```diff
@@ -502,6 +503,7 @@ def primary_key(self) -> Optional[Union[str, List[str], List[List[str]]]]:
     def __init__(self, **kwargs):
         report_granularity = kwargs.pop("report_granularity", None)
         self.attribution_window = kwargs.get("attribution_window") or 0
+        self.include_deleted = kwargs.get("include_deleted", False)
```

Comment:
looks like we can never get `True` here, please see https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/source.py#L68, another change should be added there to pass new config variables further

Reply:
done